test/user-profile-manager-test.js

```diff
@@ -64,6 +64,39 @@ describe("/lib/user-profile-manager/user-profile-manager", function () {
 			});
 		});
 
+		it("Should fail if there is a service endpoint and no version", (done) => {
+			delete process.env.VCAP_SERVICES;
+			assert.throws(() => {
+				UserProfileManager.init({
+					tenantId: "abcd",
+					throwIfFail:true,
+					appidServiceEndpoint:"zyxw"
+				});
+			}, Error, 'Failed to initialize APIStrategy. Missing version parameter');
+			done();
```

Comment:
mmm why do we have done() here? init function is synchronous

Reply:
removed the done